{
  "description": "In Lib/tarfile.py in Python through 3.8.3, an attacker is able to craft a TAR archive leading to an infinite loop when opened by tarfile.open, because _proc_pax lacks header validation.",
  "releases": {
    "bullseye": {
      "repositories": {
        "bullseye": "3.8.4~rc1-1"
      },
      "status": "open",
      "urgency": "low"
    },
    "sid": {
      "repositories": {
        "sid": "3.8.4-1"
      },
      "status": "open",
      "urgency": "low"
    }
  },
  "scope": "local"
}